pkgs/development/python-modules/lsp-tree-sitter/default.nix

```diff
@@ -14,21 +15,24 @@
 
 buildPythonPackage rec {
   pname = "lsp-tree-sitter";
-  version = "0.0.15";
+  version = "0.0.16";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "neomutt";
     repo = "lsp-tree-sitter";
-    rev = version;
-    hash = "sha256-yzScgix3BtSCBzlDoE1kMYGtVzkup/+ZK9L1C7VA3do=";
+    rev = "refs/tags/${version}";
+    hash = "sha256-nRzyVZFgb08M0I+xmhuX1LDwPsghlBLdZ2Ou8stKzk0=";
   };
 
   build-system = [
     setuptools-generate
     setuptools-scm
+    pythonRelaxDepsHook
```

Comment:
We no longer need to add pythonRelaxDepsHook manually.

> For ease of use, both buildPythonPackage and buildPythonApplication will automatically add pythonRelaxDepsHook if either pythonRelaxDeps or pythonRemoveDeps is specified.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonrelaxdepshook-using-pythonrelaxdepshook

Reply:
> We no longer need to add pythonRelaxDepsHook manually.
> 
> > For ease of use, both buildPythonPackage and buildPythonApplication will automatically add pythonRelaxDepsHook if either pythonRelaxDeps or pythonRemoveDeps is specified.
> 
> [`master`/doc/languages-frameworks/python.section.md#using-pythonrelaxdepshook-using-pythonrelaxdepshook](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md?rgh-link-date=2024-08-11T00%3A58%3A01Z#using-pythonrelaxdepshook-using-pythonrelaxdepshook)

Thanks :).